package.json

```diff
@@ -4,7 +4,9 @@
   "description": "UX Team Intern Onboarding Project",
   "main": "index.html",
   "scripts": {
-    "start": "http-server ./src -p 1820"
+    "lint": "ux-lint --fix",
+    "start": "yarn lint && http-server ./src -p 1820",
+    "build": "yarn start"
```

Comment:
this honestly would be the other way around, I'd expect `yarn start` to run `yarn lint`, then` yarn build` then the serve command (maybe make that it's own script?)

Reply:
Got it, anything specific that has to happen within build at this step in the project?

Reply:
not really, I'm not entirely sure why I included that to begin with. But here we are 🤷‍♂

Reply:
Lol ok